tests/runtime/call

```diff
@@ -436,7 +435,6 @@ EXPECT BEGIN
        @[b]: 2
        @[c]: 3
        @[d]: 4
-       
```

Comment:
Because 'avg' is now on the same path as other maps, this extra space was removed. I can add it back but wasn't sure if I should.